src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -91,23 +92,31 @@
    "No Access" #(= "NO ACCESS" (% "accessLevel"))
    })
 
-(def ^:private realm-types ["non-TCGA" "TCGA Open Access" "TCGA Protected Access"])
+(def ^:private realm-types ["Un-published" "Published" "TCGA Open Access" "TCGA Protected Access"])
+(def ^:private realm-types-defaults [true false false false])
 (def ^:private realm-predicates
- {"non-TCGA" #(not= (config/tcga-namespace) (get-in % ["workspace" "namespace"]))
+ {"Published" #(and (not= (config/tcga-namespace) (get-in % ["workspace" "namespace"])) (get-in % ["workspace" "attributes" "library:published"]))
+  "Un-published" #(and (not= (config/tcga-namespace) (get-in % ["workspace" "namespace"])) (not (get-in % ["workspace" "attributes" "library:published"])))
```

Comment:
But all the TCGA workspaces are published. so there would be no way to show published workspaces that were not TCGA. I'll run this by moran.